src/test/java/org/pentaho/metadata/automodel/PhysicalTableImporterTest.java

```diff
@@ -66,10 +66,18 @@ public void testImportTableCanExcludeColumns() throws Exception {
       importTableDefinition( database, "aSchema", "aTable", "klingon", importStrategy );
     List<IPhysicalColumn> physicalColumns = physicalTable.getPhysicalColumns();
     assertEquals( 3, physicalColumns.size() );
-    assertContainsColumn( "quantityordered", physicalColumns );
-    assertContainsColumn( "productcode", physicalColumns );
-    assertContainsColumn( "status", physicalColumns );
 
+    // Depending on the class the formatting values are set or unset. If NULL formatting values were set on the
+    // property map, this would cause an XmiParsing Error. Testing that values that are NULL are not set in the
+    // property map.
```

Comment:
It really depends on what ValueMeta type that is used. In this test I have listed what is null and then not put into the properties map. I am doing a very generic test ensures that ensures there are no nulls being set for the those formatting values that were added.

ValueMetaInteger (quantityordered) has currencySymbol set to Null
ValueMetaString (productcode and status) has mask and currencySymbol set to Null